src/views/LoginView.vue

```diff
@@ -357,24 +357,15 @@ export default {
   }
 
   .welcomeContainer h1{
-    font-size: 50px;
+    font-size: 42px;
   }
 
   .welcomeContainer p{
     font-size: 24px !important;
   }
 
   .video-container__wrapper {
-  width: 85%;
-  
-}
-}
-
-@media (orientation: landscape) and (max-width: 960px) {
-  #logo {
-    width: 64px;
-    height: 64px;
-    margin: -16px auto -32px;
```

Comment:
This caused weirdness and looked wrong:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/a16d01d8-8a2a-444f-9959-8316c2628f24)

This way the logo is a little big on mobile landscape, but still within reason

Reply:
Result:
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/ecfe0327-87c1-4e82-910a-c2ac699d4416)